app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -300,13 +315,43 @@ class SectionDateHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBi
     }
 }
 
-class SectionConflictHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBinding) :
+class SectionConflictHeaderViewHolder(private val viewBinding: ItemTxConflictSectionHeaderBinding) :
     BaseTransactionViewHolder<TransactionView.SectionConflictHeader>(viewBinding) {
 
     override fun bind(sectionDateHeader: TransactionView.SectionConflictHeader, payloads: List<Any>) {
+        val resources = viewBinding.root.context.resources
+
         with(viewBinding) {
-            sectionTitle.text = sectionDateHeader.nonce.toString()
+            nonce.text = sectionDateHeader.nonce.toString()
+            sectionTitle.text = resources.getString(R.string.tx_list_conflict_header_explainer)
+            sectionTitle.appendLink(
+                resources.getString(R.string.tx_list_conflict_header_link),
+                resources.getString(R.string.tx_list_conflict_header_learn_more)
+            )
+        }
+    }
+
+    private fun TextView.appendLink(url: String, urlText: String) {
+        val textColor = ForegroundColorSpan(textColors.defaultColor) //TODO get foreground color from TextView
+
+        val linkDrawable = ContextCompat.getDrawable(context, R.drawable.ic_link_green_24dp)!!
+        linkDrawable.setBounds(0, 0, linkDrawable.intrinsicWidth, linkDrawable.intrinsicHeight)
+        movementMethod = LinkMovementMethod.getInstance()
+        text = SpannableStringBuilder()
+            .appendText(Html.fromHtml(text.toString()), textColor)
+            .append(" ")
+            .appendText(urlText, URLSpan(url))
+            .append(" ")
+            .appendTextWithSpans(" ", listOf(ImageSpan(linkDrawable, ImageSpan.ALIGN_BASELINE), URLSpan(url)))
+    }
+
+    private fun SpannableStringBuilder.appendTextWithSpans(text: CharSequence, what: List<Any>, flags: Int = 0): SpannableStringBuilder {
+        val start = length
```

Comment:
Wouldn't you have to update `start` for each item in `what`?

Reply:
I don't think I understand very well what this method does. Could you explain it to me?

Reply:
> Wouldn't you have to update start for each item in what?

No. Because the whats are supposed to be on top of each other. In this case the Image should also be clickable and open the url. Other cases could be a list of underline and color change. Both need to be applied to the same range.